deploy/dependencies.yaml

```diff
@@ -640,53 +970,72 @@ metadata:
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: installs.tekton.dev
+  name: config.operator.tekton.dev
 spec:
-  group: tekton.dev
+  group: operator.tekton.dev
   names:
-    kind: Install
-    listKind: InstallList
-    plural: installs
-    singular: install
-  scope: Namespaced
+    kind: Config
+    listKind: ConfigList
+    plural: config
+    singular: config
+  scope: Cluster
   subresources:
     status: {}
   validation:
     openAPIV3Schema:
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
           type: string
         kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
         metadata:
           type: object
         spec:
+          properties:
+            targetNamespace:
+              description: namespace where OpenShift pipelines will be installed
+              type: string
+          required:
+          - targetNamespace
           type: object
         status:
           properties:
-            resources:
-              description: The resources applied
+            conditions:
+              description: installation status sorted in reverse chronological order
               items:
-                type: string
+                properties:
+                  code:
+                    description: Code indicates the status of installation of pipeline resources.
+                    type: string
+                  details:
+                    description: Additional details about the Code
+                    type: string
+                  version:
+                    description: The version of tekton pipelines
+                    type: string
+                required:
+                - code
+                - version
+                type: object
               type: array
-            version:
-              type: string
-          required:
-          - resources
-          - version
           type: object
+  additionalPrinterColumns:
+  - JSONPath: ".status.conditions[0].code"
+    name: status
+    type: string
   version: v1alpha1
   versions:
   - name: v1alpha1
     served: true
     storage: true
 ---
+apiVersion: operator.tekton.dev/v1alpha1
+kind: Config
```

Comment:
My only real issue here is that there is code in the kabanero controller that tries to create an instance of the old `Install` CRD if it did not exist before.  
https://github.com/kabanero-io/kabanero-operator/blob/master/pkg/controller/kabaneroplatform/tekton.go

It seems like this code is not necessary anymore, and should be deleted, or at least verify that the `Config` object exists and log a message if it's not.

The same is also probably true of the knative-eventing and knative-serving code in the kabanero controller, as I'm not sure those objects are necessary anymore either.

I'd like to hear @kyleschlosser's comments as well as most of this code pre-dates my involvement in this project.

Reply:
@dacleyra this back levels the serving and eventing operators. Can you explain why that is? 

Regarding the Tekton install CRD, we need to do two things: 
1. Update the sources so that the removed resource kind is not created or attempted
2. Remove any remaining install resource kind which we created in a prior version of the operator

Reply:
version 0.6.0 is used instead of v0.7.x due to eventing-contrib githubsources being broken
https://github.com/knative/eventing-contrib/issues/481

The resolution to the issue has only just made it into the v0.8.0 release.

eventing-contrib 0.6 doesn't work with eventing & serving 0.7 as an option either.

Holding off to pick up newer releases, 0.7 or 0.8, dependencies and requirements are changed
https://github.com/knative/serving/releases